content/docs/command-reference/status.md

```diff
@@ -56,32 +59,32 @@ This indicates that no differences were detected, and therefore no stages would
 be executed by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the changes in <abbr>dependencies</abbr>
+`.dvc` file (stage) with differences, the changes in <abbr>dependencies</abbr>
 and/or <abbr>outputs</abbr> that differ are listed. For each item listed, either
 the file name or hash is shown, and additionally a status word is shown
 describing the changes (described below).
 
-- _changed checksum_ means that the <abbr>DVC-file</abbr> hash has changed (e.g.
-  someone manually edited the file).
+- _changed checksum_ means that the <abbr>`.dvc` file</abbr> hash has changed
+  (e.g. someone manually edited the file).
 
-- _always changed_ means that this is a DVC-file with no dependencies (an
+- _always changed_ means that this is a `.dvc` file with no dependencies (an
   _orphan_ stage file) or that it has the `always_changed: true` value set (see
   `--always-changed` option in `dvc run`), so its considered always changed, and
   thus is always executed by `dvc repro`.
 
 - _changed deps_ or _changed outs_ means that there are changes in dependencies
-  or outputs tracked by the <abbr>DVC-file</abbr>. Depending on the use case,
+  or outputs tracked by the <abbr>`.dvc` file</abbr>. Depending on the use case,
   commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to update
   the file. Possible states are:
 
   - _new_: An <abbr>output</abbr> is found in the workspace, but there is no
-    corresponding file hash saved in a DVC-file yet.
+    corresponding file hash saved in a `.dvc` file yet.
   - _modified_: An output or <abbr>dependency</abbr> is found in the workspace,
-    but the corresponding file hash the DVC-file is not up to date.
-  - _deleted_: The output or dependency is references in a DVC-file, but does
+    but the corresponding file hash the `.dvc` file is not up to date.
```

Comment:
but the corresponding file hash the `dvc.lock` or `.dvc` file

Reply:
I changed this to:

> but the corresponding file hash in the `dvc.lock` or `.dvc` file

Is that okay?

Reply:
Yes thanks I didn't write it correctly. I think we have an extra "the" now though, I'll open a separate comment 🙂